content/the-ember-times-issue-176.md

```diff
@@ -31,15 +33,13 @@ Here at the Ember Times, we are super excited about the conference [lineup](http
 
 ---
 
-## [2. Section title in sentence case 🐹](section-url)
+## [The ins and outs of organising EmberConf 📻](https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+In the latest [Ember Report](https://twitter.com/EmberReport) [podcast]((https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021)) [Leah Silber (@wifelette)](https://twitter.com/wifelette) talks about organising EmberConf remote, the history of EmberConf and the topics like [**inclusion**](https://emberconf.com/about/inclusiveness-at-emberconf) that are so crucial to the EmberConf vibe.
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+This podcast gives a nice insight in the mind of a great conference organiser with some tips and pointers for others looking to organise conferences or meet-ups remote and in real live.
+
+Besides information on how to organise a conference Leah also tells us about her favourite talks and the cool things that will happen this EmberConf including workshops and the [kids track](https://emberconf.com/kids).
```

Comment:
```suggestion
In addition to how to organise a conference, Leah tells us about her favourite talks and the cool things that will happen at EmberConf 2021, such as workshops and the [kids track](https://emberconf.com/kids).
```

After this paragraph, what do you think about adding a call to action? Something like:

```
Be sure to check out the podcast episode, [Leah Silber on EmberConf 2021](https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021)!
```